### PR TITLE
bootswatch4 beta

### PR DESCRIPTION
**Pull request by jmervine**
Addressing https://github.com/MaxCDN/bootstrap-cdn/issues/881

- Updating download scripts to support bootswatch 4.
- Adding new route.
- Adjusting beta placement in header (@jdorfman thoughts?)
- Adding bootswatch4 files.
- Fixing bug with bootswatch 3 images showing bootswatch 4 thumbnails.
- Fixing bug with bootswatch 3 links linking to bootswatch 4.

### CODE REVIEW

**Comment by jmervine** [comment]
@jdorfman I have not uploaded the files to s3 yet. Wanted to get your feedback on this first.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 web:
-  image: node:8
+  image: jmervine/herokudev-node:8
```
Comment (XhmikosR) [inline docker-compose.yml]: What's this change about?

File: scripts/bootswatch.js [hunk 2]
```diff
@@ -7,7 +7,9 @@ const path    = require('path');
 const fs      = require('fs');
 const request = require('request');
 
-const version = process.argv[2];
+const version  = process.argv[2];
+const verMajor = version[0];
```
Comment (XhmikosR) [inline scripts/bootswatch.js]: Can't we just do this programmatically instead of adding a new arg?

Comment (jmervine) [reply scripts/bootswatch.js]: It's not a new arg, it's taking the first character of the string.

File: docker-compose.yml (hunk 1 above)
Comment (jmervine) [reply docker-compose.yml]: Just using a different base image for my local development environment. One that more closely matches the deployment target. This doesn't impact the application running on heroku.

Comment (jmervine) [reply docker-compose.yml]: If you run docker, your can use with the following command:

```
docker-compose run --rm --service-ports web bash

# then setup with
npm install

# and run with
node app.js

# or run tests with
node make test
# ... etc
```

File: scripts/bootswatch.js (hunk 2 above)
Comment (XhmikosR) [reply scripts/bootswatch.js]: True, sorry, too tired :/

**Comment by jmervine** [comment]
@jdorfman Cherry picked in your commit mentioned here:  https://github.com/MaxCDN/bootstrap-cdn/issues/881#issuecomment-346442293

**Pull request merged**